plugins/inputs/procstat/nativeFinder_windows.go

```diff
@@ -0,0 +1,109 @@
+// +build windows
+
+package procstat
+
+import (
+	"context"
+	"fmt"
+	"regexp"
+	"time"
+
+	"github.com/StackExchange/wmi"
+	"github.com/shirou/gopsutil/process"
+)
+
+//Timeout is the timeout used when making wmi calls
+var Timeout = 5 * time.Second
+
+func init() {
+	wmi.DefaultClient.AllowMissingFields = true
```

Comment:
We should try to arrange for a new wmi.Client to be created and used once for each plugin.  This will avoid needing to modify the default client which might have unwanted effects on other pieces of code due to it being global.

Reply:
should I have a seperate native finder struct for windows so that I can store it there?

Reply:
I was able to remove this and my queries still work. I took this from the gopsutil implementation and it looks like it's not really needed for our queries.

Reply:
:+1: